### PR TITLE
MVC items + Navbar

### PR DESCRIPTION
**Pull request by Alexs1004**
comme onnnne baaabyyyybabbyy

### CODE REVIEW

File: app/assets/stylesheets/items/_new.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.button {
+  text-align: center;
+  margin-bottom: 50px;
+  margin-top: 50px;
+}
+
+.toto {
```
Comment (Alexs1004) [inline app/assets/stylesheets/items/_new.scss:7]: 💩

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+
+  <%# if user_signed_in? %>
+
+
```
Comment (Alexs1004) [inline app/views/shared/_navbar.html.erb:4]: Empty lines

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -8,10 +8,19 @@
...
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
 
   <body>
-    <%= yield %>
+    <button type="button" class="btn btn-link mb-2" data-toggle="modal" data-target="#exampleModal">
+      Launch demo modal
```
Comment (Alexs1004) [inline app/views/layouts/application.html.erb:20]: remove

File: app/views/items/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<div class="top-page">
+</div>
+
+<div class="card-product">
+  <img src=" <%= asset_url("showbanner.jpg") %>" alt="castle">
```
Comment (Alexs1004) [inline app/views/items/show.html.erb:5]: use image_tag

File: Gemfile.lock [hunk 5]
```diff
@@ -276,7 +274,6 @@ GEM
     zeitwerk (2.4.2)
 
 PLATFORMS
-  ruby
```
Comment (hadri96) [inline Gemfile.lock]: You can't erase that

**Review by hadri96 — CHANGES_REQUESTED** [review]

**Review by hadri96 — APPROVED** [review]

**Pull request merged**